### PR TITLE
fix: Rehaul Supervisor

### PR DESCRIPTION
**Pull request by jnikhila**
- Restructuring
- Rewriting

### CODE REVIEW

File: website/docs/getting-started/setup/instance-management/supervisor.md [hunk 1]
```diff
@@ -3,28 +3,45 @@ sidebar_position: 1
 ---
 # Supervisor
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-management/supervisor.md:4]:
```suggestion
# Monitor Processes
```

File: website/docs/getting-started/setup/instance-management/supervisor.md [hunk 2]
```diff
@@ -3,28 +3,45 @@ sidebar_position: 1
...
-Supervisor comes with a web interface for managing the various processes, available at [http://localhost/supervisor](http://localhost/supervisor), as well as a command-line interface towards the same goal.
+[Supervisor](http://supervisord.org/) provides:
+* [Web Interface](#web-interface)
+* [Command line Interface](#command-line-interface) 
 
-:::info
-The credentials for supervisor access are defined by the environment variables `APPSMITH_SUPERVISOR_USER` and `APPSMITH_SUPERVISOR_PASSWORD` in the `stacks/configuration/docker.env` file.
-:::
+## Web interface
+The Supervisor web interface can be accessed at `http://localhost/supervisor`.
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-management/supervisor.md:13]: this line and the image ideally should be combine with step 3 so the user know they have to set it up first and then access it

File: website/docs/getting-started/setup/instance-management/supervisor.md [hunk 3]
```diff
@@ -3,28 +3,45 @@ sidebar_position: 1
...
-The command-line interface can also be used to perform operations like restarting the Appsmith server, restarting Nginx, etc. For example, the following command (run in the installation folder) can be used to get the status of all running processes:
+```bash
+APPSMITH_SUPERVISOR_USER=<SUPERVISOR-USER>
+APPSMITH_SUPERVISOR_PASSWORD=<SUPERVISOR-PASSWORD>
+```
+
+3. Use the interface to manage the Appsmith server, Nginx, MongoDB, and more.
+
+## Command line interface
+The command line interface also helps in managing the Appsmith server, Nginx, MongoDB, and more.
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-management/supervisor.md:32]: Make sure we confirm that env variables need not be set up use the command line interface.

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/setup/instance-management/supervisor.md (hunk 3 above)
Comment (sum35h) [reply website/docs/getting-started/setup/instance-management/supervisor.md:31]: The credentials through the env vars are not required for supervisorctl CLI. They are only required by browser

**Comment by sum35h** [comment]
@jnikhila We will need to update the Supervisor UI screenshot with the latest one, since there are a lot more processes added now.

**Review by sum35h — APPROVED** [review]

**Comment by jnikhila** [comment]
> @jnikhila We will need to update the Supervisor UI screenshot with the latest one, since there are a lot more processes added now.

As discussed when the feature will be out, we can update the screenshot. For now, we'll move ahead with this.

**Review by danciaclara — APPROVED** [review]

**Pull request merged**